darwin/dataset.py

```diff
@@ -101,7 +101,7 @@ def _add_files_to_dataset(self, filenames: List, fps: int):
     def pull(self, image_status: str = None):
```

Comment:
the type would then be `image_status: Optional[str]`

Reply:
Didn't know about `Optional`. Fixed.